pandas/tests/groupby/test_groupby.py

```diff
@@ -5777,6 +5769,81 @@ def test_agg_over_numpy_arrays(self):
 
         assert_frame_equal(result, expected)
 
+    def test_cummin_cummax(self):
+        # GH 15048
+        # Test with different dtypes
+        for dtype in [np.int32, np.float32, np.float64]:
```

Comment:
test with int64, datetime types *should* work as well (e.g. datetime64, timedelta64).